### PR TITLE
[DESIGN2-207-Popover] Popover Component - DSCO - Update color variables

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with Primitive colors from `primitiveColors.css` on the Popover component.

👀 **DOTD note:** there might be Eyes diffs with this.

## Links
Jira ticket: [DESIGN2-207](https://codedotorg.atlassian.net/browse/DESIGN2-207)

## Testing story
Tested locally in Storybook

----

## Before - using `shared/colors.scss`
Before

## After - using `primitiveColors.css`
After

### CODE REVIEW

File: apps/src/componentLibrary/popover/popover.module.scss [hunk 1]
```diff
@@ -84,7 +84,7 @@ $m-tail-length: 12px;
     }
 
     .iconSection {
-      color: $light_secondary_500;
+      color: var(--brand-purple-50);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/popover/popover.module.scss:87]: Screenshot 2024-12-11 at 2 28 18 PM

File: apps/src/componentLibrary/popover/popover.module.scss [hunk 2]
```diff
@@ -70,7 +70,7 @@ $m-tail-length: 12px;
   flex-direction: column;
   justify-content: flex-end;
   align-items: center;
-  background-color: $light_white;
+  background-color: var(--neutral-base-white);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/popover/popover.module.scss:73]: Screenshot 2024-12-11 at 2 29 57 PM

**Comment by kelbyhawn** [comment]
@levadadenys I'm only showing screenshots of the icon popover since it has the most color updates.

**Review by levadadenys — APPROVED** [review]
LGTM! 🚀

**Pull request merged**